### DISCUSSION TITLE
Image creation, export and parameter recognition issues

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1370] [Questions & Problems]
1.Can a fold error line be added to the comparison of predicted values with observed values in the parameter identification module？（as shown in the following figure，from：Zheng L, Tang S, Tang R, Xu M, Jiang X, Wang L. Dose Adjustment of Quetiapine and Aripiprazole for Pregnant Women Using Physiologically Based Pharmacokinetic Modeling and Simulation. Clin Pharmacokinet. 2021;60(5):623-635. doi:10.1007/s40262-020-00962-3）
![19903a196fecbbc37e1c039e2540211](https://user-images.githubusercontent.com/86552551/199056756-fde64a33-27ed-4f2c-ad8a-7fda1c9af89f.jpg)

2.How to change the color shade of the exported image's coordinate axes？
![dc664b84400f4ed21915f78573baa7a](https://user-images.githubusercontent.com/86552551/199057200-2881b944-41ce-42f6-9a77-42c57362c6f1.png)
![3d5cd9fdac2d2369cd01b2e5ed2d3fe](https://user-images.githubusercontent.com/86552551/199057252-64932968-73e2-473f-924e-73fa6fd0da1a.png)

3.How to perform the operation calculate sensitivity under the confidence interval in the parameter identification module？
![a2f9493bae8dee37cb746a620a3e8f6](https://user-images.githubusercontent.com/86552551/199057576-272c809c-e969-42cc-b227-b74cbd4781e2.png)

Best regards，
SCZ

**Comment by StephanSchaller** [comment]
Dear @SCZ1174 , 

1. Yes, this is currently in development
2. you need to activate the checkbox "calculate sensitivities" in the "Configuration" tab.

BEst, Stephan
